src/main/java/Duke.java

```diff
@@ -1,10 +1,114 @@
+        Scanner in = new Scanner(System.in);
+        while (!isExited) {
+            String line;
+            line = readInput(in);
+            isExited = evaluateInput(line);
+        }
+    }
+
+    public static void printGreetingMessage () {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static String readInput (Scanner in) {
+        return in.nextLine();
+    }
+
+    public static boolean evaluateInput (String line){
+        String upperLine = line.toUpperCase();
+        if (upperLine.startsWith("TODO")) {
+            addTask(new Todo(line));
+            return false;
+        } else if (upperLine.startsWith("LIST")) {
+            showList();
+            return false;
+        } else if (upperLine.startsWith("DEADLINE")) {
+            addDeadline(line);
+            return false;
+        } else if (upperLine.startsWith("EVENT")) {
+            addEvent(line);
+            return false;
+        } else if (upperLine.startsWith("BYE")) {
+            printByeMessage();
+            return true;
+        } else if (upperLine.startsWith("DONE")) {
+            markTaskAsDone(line);
+            return false;
+        } else {
+            printInvalidInputMessage();
+            return false;
+        }
+    }
+
+    public static void printInvalidInputMessage() {
+        System.out.println("Invalid Input");
+    }
+
+    public static void markTaskAsDone(String line) {
+        String[] words = line.split(" ");
+        int taskNum = Integer.parseInt(words[1]);
+        tasks[taskNum - 1].markAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("\t" + tasks[taskNum - 1]);
+    }
+
+    public static void addEvent(String line) {
+        int slashPosition  =  line.indexOf("/");
+        String at = line.substring(slashPosition + 1);
+        String description = line.substring(EVENT_LENGTH,slashPosition);
+        addTask(new Event(description,at));
+    }
+
+    public static void addDeadline(String line) {
+        int slashPosition  =  line.indexOf("/");
+        String by = line.substring(slashPosition + 1);
+        String description = line.substring(DEADLINE_LENGTH,slashPosition);
+        addTask(new Deadline(description,by));
+    }
+
+    public static void addTask (Task t){
+        System.out.println("Got it. I've added this task:");
+        tasks[numOfTask] = t;
+        numOfTask++;
+        System.out.println("\t" + t);
+        System.out.println("Now you have " + numOfTask + " task in the list.");
+    }
+
+    public static void showList () {
+        int numbering = 0;
```

Comment:
Naming of variable is abit unclear here.
Suggestion: Make this variable to be name more to its function , more towards what it is being used.